### PR TITLE
Fix/title-type-in-name-search

### PR DESCRIPTION
**Pull request by MatheusFrate**
# 🎬 Filmaro 🎬 - Pull Request

## Description:

Fixed the title type during name search

## Type of change:

- [ ] New feature (This PR adds a new functionality to the project)
- [ ] Technical debt (This PR improve features already created)
- [x] Bug fix (This PR covers solving a bug)
- [ ] Documentation (This PR require a documentation update, i.e. the class doc)
- [x] Issue resolution (This PR solve an github open issue)
[Link to solved Issue](https://github.com/Filmaro/Filmaro/issues/36 )

## Checklist:

- [x] My changes do not generate a new warning or issue.
- [x] I deleted all the code lines that are not being used.
- [x] I commented my code in some areas i believed that are difficult to understand.

👩🏽‍💻~ Filmaro Team ~👨🏽‍💻

### CODE REVIEW

File: src/components/FilterTitles/FilterTitles.js [hunk 1]
```diff
@@ -18,7 +18,6 @@ export const FilterTitles = () => {
   useEffect(() => {
```
Comment (JoaoWudarski) [inline src/components/FilterTitles/FilterTitles.js:19]: Antes de tudo, 3 coisas:

1.  Muda o nome do PR, ele entrou como o nome do commit kkkk
2. Altera a branch de destino para develop. (Inclusive vou alterar para a padrão ser a develop)
3. Coloca as labels... Acho que nesse caso é minor e bug.

**Review by Im0tec — APPROVED** [review]
Só arrumar os detalhes que o Wu comentou ai e mandar bala no merge

File: src/components/FilterTitles/FilterTitles.js (hunk 1 above)
Comment (MatheusFrate) [reply src/components/FilterTitles/FilterTitles.js:19]: osh, eu tinha colocado as labels antes kkkk
e pq que entrou com o nome do commit uai

Comment (MatheusFrate) [reply src/components/FilterTitles/FilterTitles.js:19]: mudei

Comment (JoaoWudarski) [reply src/components/FilterTitles/FilterTitles.js:19]: SóAgradece

**Review by JoaoWudarski — APPROVED** [review]
Taque duro teteuzada

**Pull request merged**